### PR TITLE
common-instancetypes: Add release-0.3 presubmits

### PR DESCRIPTION
**Pull request by lyarwood**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-0.3.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+presubmits:
+  kubevirt/common-instancetypes:
+  - name: pull-common-instancetypes
+    branches:
+      - release-0.3
+    always_run: true
+    optional: true
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-0.3.yaml:7]: Are you sure you want this as optional?

Comment (lyarwood) [reply github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-0.3.yaml:7]: That's the behaviour on main at the moment, I'll confirm there and will backport a change to this branch if we can drop it.

**Pull request merged**